tasks/gcp/StatisticalPhasingTasks.wdl

```diff
@@ -85,11 +87,76 @@ task ShapeIt4 {
   }
 
   output {
-      File phased_vcf =  "~{project_id}_phased.vcf.gz"
+      File region_phased_vcf =  "~{output_filename}"
       File log_file =  "phased.log"
   }
 }
 
+
+
+task LigateRegions {
+  input {
+    Array[File] region_phased_vcfs
+    Array[File] region_phased_vcfs_indices
+    File interval_list
+    String project_id
+
+    String docker_tag = "us.gcr.io/broad-gotc-prod/malariagen/bcftoolspython:1.11"
+    Int preemptible_tries = runTimeSettings.preemptible_tries
+    Int num_cpu = 1
+    RunTimeSettings runTimeSettings
+  }
+
+  Int disk_size = ceil(size(region_phased_vcfs, "GiB")) * 2 + 20
+
+  command {
+    set -e pipefail
+
+    python3 <<CODE
+    from pathlib import Path
+
+    index_files = [ "~{sep='", "' region_phased_vcfs_indices}" ]
+    for f in index_files:
+        Path(f).touch()
```

Comment:
Do you still need this touch?

Reply:
Probably don't need this, but it's working and not giving warnings, so I am going to leave it for now